documentation/release_checklist_projectname.md

```diff
@@ -0,0 +1,165 @@
+# Release Checklist Android
+
+## Project Name
+#### Person who is responsible for the release
+
+## General
+| Task | Done | Comment |
+|---|---|---|
+|Update GIT|||
+|Refresh project|||
+|Clean project|||
+
+## Manifest
```

Comment:
evtl. Wording anpassen: wird ja mittlerweile alles in der build.gradle gehandhabt